lexers/f/fortran_fixed.go

```diff
@@ -0,0 +1,39 @@
+package f
+
+import (
+	. "github.com/alecthomas/chroma" // nolint
+	"github.com/alecthomas/chroma/lexers/internal"
+)
+
+// FortranFixed lexer.
+var FortranFixed = internal.Register(MustNewLazyLexer(
+	&Config{
+		Name:            "FortranFixed",
+		Aliases:         []string{"fortranfixed"},
+		Filenames:       []string{"*.f", "*.F"},
+		MimeTypes:       []string{"text/x-fortran"},
```

Comment:
I don't know what FortranFixed is, but in hindsight it's weird that `.f` doesn't match the fortran lexer?